src/main/java/pl/edu/agh/firecell/model/Cell.java

```diff
@@ -5,16 +5,21 @@ public record Cell (
         int burningTime,
         boolean flammable,
         Material material,
-        int remainingFirePillar
+        int remainingFirePillar,
+        double smokeIndicator
         ) {
 
```

Comment:
Hit `ctrl + alt + L` inside that file, there are some problems with tabs and spaces :P